lib/config.js

```diff
@@ -6,36 +6,20 @@ const path = require('path')
 
 module.exports = {
   AWS_INSTANCE_CHECK_URL: 'http://169.254.169.254/latest/dynamic/instance-identity/document/'
-, BUF_LIMIT: process.env.BUF_LIMIT || 10000 // 10000 lines
-, COMPRESS: isNaN(process.env.COMPRESS) ? true : +process.env.COMPRESS
+, COMPRESS: isNaN(process.env.COMPRESS) || process.env.COMPRESS === '1'
 , DEFAULT_CONF_FILE: os.platform() !== 'win32' ? '/etc/logdna.conf' : path.join(process.env.ALLUSERSPROFILE, '/logdna/logdna.conf')
-, DEFAULT_HTTP_ERRORS: ['ECONNRESET', 'EHOSTUNREACH', 'ETIMEDOUT', 'ESOCKETTIMEDOUT', 'ECONNREFUSED', 'ENOTFOUND']
 , DEFAULT_LOG_PATH: os.platform() !== 'win32' ? '/var/log' : path.join(process.env.ALLUSERSPROFILE, '/logs')
-, DEFAULT_REQ_HEADERS: {'Content-Type': 'application/json; charset=UTF-8'}
-, DEFAULT_REQ_HEADERS_GZIP: {'Content-Type': 'application/json; charset=UTF-8', 'Content-Encoding': 'gzip'}
 , DEFAULT_WINTAIL_FILE: os.platform() !== 'win32' ? '/etc/winTail.ps1' : path.join(process.env.ALLUSERSPROFILE, '/logdna/winTail.ps1')
 , FILEQUEUE_PATH: os.platform() !== 'win32'
     ? (process.env.FILEQUEUE_PATH || '/tmp')
     : path.join(process.env.ALLUSERSPROFILE, (process.env.FILEQUEUE_PATH || '/tmp'))
-, FLUSH_INTERVAL: process.env.FLUSH_INTERVAL || 250 // 250 millisec
-, FLUSH_LIMIT: process.env.FLUSH_LIMIT || 5000 // 5000 lines
-, GZIP_COMPRESS_LEVEL: process.env.GZIP_COMPRESS_LEVEL || 2
-, HEALTHCHECK_INTERVAL: process.env.HEALTHCHECK_INTERVAL || 1800000 // 30 minutes
-, HTTP_KEEPALIVE: process.env.HTTP_KEEPALIVE || 60000 // 1 min
-, HTTP_RETRY: isNaN(process.env.HTTP_RETRY) ? true : +process.env.HTTP_RETRY
-, HTTP_TIMEOUT: process.env.HTTP_TIMEOUT || 30000 // 30 sec
-, LOGDNA_HEALTHCHECK: process.env.LDHEALTHCHECK || process.env.HEALTHCHECK_ENDPOINT || '/healthcheck'
-, LOGDNA_LOGENDPOINT: process.env.LDLOGPATH || process.env.INGESTION_ENDPOINT || '/logs/agent'
-, LOGDNA_LOGHOST: process.env.LDLOGHOST || process.env.INGESTION_HOST || 'logs.logdna.com'
-, LOGDNA_LOGPORT: process.env.LDLOGPORT || process.env.INGESTION_PORT || 443
-, LOGDNA_LOGSSL: isNaN(process.env.LDLOGSSL) ? true : +process.env.LDLOGSSL
-, LOGDNA_RECONNECT: true
-, MAX_SOCKETS: process.env.MAX_SOCKETS || 20
+, FLUSH_INTERVAL: process.env.FLUSH_INTERVAL
+, FLUSH_LIMIT: process.env.FLUSH_LIMIT
+, PROXY: process.env.HTTPS_PROXY || process.env.HTTP_PROXY
+, LOGDNA_INGESTION_URL: process.env.LOGDNA_INGESTION_URL || 'https://logs.logdna.com/logs/agent'
```

Comment:
Sorry for nitpicking on naming but what about keeping the config variable / parameter name consistent across the integrations? Since we are removing all `LOGDNA_*` parameters, why to call this `LOGDNA_INGESTION_URL`?

On `@logdna/logger-node`, it's `url`; on `logspout` and lambdas, it's `LOGDNA_URL` since they are also environment variables there - as you mentioned, naming is hard but I'd suggest to call it `LOGDNA_URL`.

Reply:
ok, that's fine